plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/nexus/ldap/internal/LdapTestSupport.java

```diff
@@ -172,7 +172,13 @@ protected synchronized void resumeLdapServer(final String name) throws Exception
 
   @After
   public void stopLdap() throws Exception {
-    lookup(CacheManager.class).shutdown();
+    try {
+      lookup(CacheManager.class).close();
+    }
+    catch (Exception e) {
+      util.getLog().warn("Failed to shutdown cache-manager", e);
```

Comment:
parity with other test support close, /me shrugs unsure if this even throws an exception anymore, but for these 2 odd places doing injection for testing I don't really care too much.